### DISCUSSION TITLE
Why are there jQuery dependencies in tom-select node_modules?

### DISCUSSION THREAD

**Discussion by atstockland** [orchidjs/tom-select#207] [General]
I thought I'd give tom-select a try as I'm hunting for a nice searchable select that doesn't use jQuery.  I'm trying to migrate my app away from any unnecessary dependencies.  This gem states "decoupling from jQuery".

I pushed some updates out to Heroku, which included the addition of tom-select...which I added via yarn "tom-select": "^2.0.0-rc.1".  My heroku slug size jumped up a bit and decided to investigate why.  To my surprise... node_modules/tom-select/dist/docs/js includes a 436 kb jqueryui.js file and a 97 kb jQuery.min.js file.  The overall size for tom-select is 10.5 megs.

Why are there jQuery dependencies in there?  Thanks!

**Comment by atstockland** [comment]
Sorry, this would not change the size of my heroku slug because node_modules are ignored in git.  So, unless I am mistaken, this does not account for slug size...but, it still begs the question why jQuery is in there?  Perhaps, that has nothing to do with tom-select.  I'm doing some homework on this...

**Reply by oyejorge** [reply]
The generated contents of the documentation website (tom-select.js.org) are currently in the  ```node_modules/tom-select/dist/docs``` folder. I'll change that because there's no need for those to be there.
